### ISSUE TITLE
plcalc serialize does not handle extended (declare) expressions correctly

### ISSUE THREAD

**Issue by dahersh** [mbleron/oracle#11]
` plcalc.serialize(plcalc.compile('declare(x:=1,y:=2) return x/y' ))` 
yields:
` X := 1`

**Comment by mbleron** [comment]
Thanks for raising this issue. Indeed, I completely overlooked that part.
I'll try to fix it ASAP.

**Comment by mbleron** [comment]
Fixed in v2.0.
Important :  I had to modify the internal form of the compiled expression, rendering the new version incompatible with previous ones.
So If you were using **stored** compiled expressions containing extended constructs, you'll have to recompile them from their plain text representations, using the new version.
Expressions free of extended constructs are unaffected.

```
select plcalc.serialize(
         plcalc.compile(
           'declare (z := (declare (x:=1, y:=2) return x+y) )
            return z^2 - 2*z + 1'
         )
       ) as result
from dual;
 
RESULT
--------------------------------------------------------------------------------
DECLARE (Z := (DECLARE (X := 1,Y := 2) RETURN X + Y)) RETURN Z ^ 2 - 2 * Z + 1
```

**Comment by dahersh** [comment]
Thank you very much, Marc!
